exercises/sublist/description.md

```diff
@@ -1,18 +1,24 @@
 # Description
 
-Given two lists determine if the first list is contained within the second
-list, if the second list is contained within the first list, if both lists are
-contained within each other or if none of these are true.
+Given any two lists `A` and `B`, determine if:
 
-Specifically, a list A is a sublist of list B if by dropping 0 or more elements
-from the front of B and 0 or more elements from the back of B you get a list
-that's completely equal to A.
+* List `A` is equal to list `B`; or
+* List `A` contains list `B` (`A` is a superlist of `B`); or
+* List `B` contains list `A` (`A` is a sublist of `B`); or
```

Comment:
Perhaps:
```none
* List `A` is contained by list `B` (`A` is a sublist of `B`); or
```
keeping A consistently before B would aid comprehension, I think

Reply:
@wolf99 I would lean towards my original suggestion, if it was up to me. Just don't want to waste too much of your time :).

Reply:
I don't feel that either should block this PR from being merged.
Just a counterpoint to the suggestion from @glennj.
Perhaps Glenn say if my argument has swayed his opinion; if it has not I'm OK if you go with his suggestion if you agree.